package.json

```diff
@@ -43,7 +44,9 @@
     "ethereumjs-util": "^6.0.0",
     "fast-memoize": "^2.5.1",
     "merkletreejs": "0.0.22",
-    "openzeppelin-solidity": "2.1.1"
+    "mongodb": "^3.1.13",
```

Comment:
should this be devDependency?

Reply:
Im not sure. It is necessary for the application to run.

Reply:
No I think this is a standard dependency for the project to work. The dev-dependencies, are for things the travis build needs to run (i.e. like for the linter etc..)